src/libraries/System.IO.Ports/ref/System.IO.Ports.csproj

```diff
@@ -1,9 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
   </PropertyGroup>
+
   <ItemGroup>
     <Compile Condition="'$(TargetFramework)' != 'net461'" Include="System.IO.Ports.cs" />
     <Compile Condition="'$(TargetFramework)' == 'net461'" Include="System.IO.Ports.net461.cs" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+    <ProjectReference Include="$(LibrariesProjectRoot)System.ComponentModel.Primitives\ref\System.ComponentModel.Primitives.csproj" />
```

Comment:
what do you need ComponentModel for?

Reply:
It derives from Component: https://github.com/dotnet/runtime/blob/709fcd80586aaca02cf41069d951612c0318a95b/src/libraries/System.IO.Ports/ref/System.IO.Ports.cs#L63.

Reply:
fine then I guess